src/main/java/io/airlift/airline/TypeConverter.java

```diff
@@ -54,7 +55,11 @@ else if (Double.class.isAssignableFrom(type) || Double.TYPE.isAssignableFrom(typ
                 return valueOf.invoke(null, value);
             }
         }
-        catch (Throwable ignored) {
+        catch (InvocationTargetException ex) {
+            throw new ParseOptionConversionException(name, value, type.getSimpleName(), ex.getTargetException());
+        }
+        catch (IllegalAccessException |
+                NoSuchMethodException ignored) {
```

Comment:
please wrap on previous line (same below)

Reply:
Instead if `IllegalAccessException | NoSuchMethodException` use just `ReflectiveOperationException`

Reply:
We already had that discussed at https://github.com/airlift/airline/pull/68#discussion_r347261254